python/pyspark/worker.py

```diff
@@ -50,7 +50,7 @@
     ArrowStreamUDFSerializer,
     ApplyInPandasWithStateSerializer,
 )
-from pyspark.sql.pandas.types import to_arrow_type
+from pyspark.sql.pandas.types import _create_row, to_arrow_type
```

Comment:
nit: `_create_row` should be imported from `pyspark.sql.types`

Reply:
Done.